src/standards.jl

```diff
@@ -70,33 +72,43 @@ const STANDARD_LABELS = Dict(# This EEG channel name list is a combined 10/20 an
                                                 "v1", "v2", "v3", "v4", "v5", "v6", "v7", "v8", "v9",
                                                 "v1r", "v2r", "v3r", "v4r", "v5r", "v6r", "v7r", "v8r", "v9r",
                                                 "x", "y", "z"],
-                             ["eog"] => ["left"=> ["eogl", "loc", "lefteye", "leye", "e1", "l"],
-                                         "right"=> ["eogr", "roc", "righteye", "reye", "e2", "r"]],
-                             ["emg"] => ["chin1", "chin2", "chin3",
+                             # EOG should not have any channel names overlapping with EEG channel names
+                             ["eog", "eeg"] => ["left"=> ["eogl", "loc", "lefteye", "leye", "e1", "eog1", "l", "left eye", "leog", "log", "li"],
+                                                "right"=> ["eogr", "roc", "righteye", "reye", "e2", "eog2", "r", "right eye", "reog", "rog", "re"]],
+                             ["emg"] => ["chin1" => ["chn", "chin_1", "chn1", "kinn", "menton", "submental", "submentalis", "submental1", "subm1", "chin", "mentalis", "chinl", "chinli", "chinleft", "subm_1", "subment"],
+                                         "chin2" => ["chn2", "chin_2", "submental2", "subm2", "chinr", "chinre", "chinright", "subm_2"],
+                                         "chin3" => ["chn3", "submental3", "subm3", "chincenter"],
                                          "intercostal"=> ["ic"],
-                                         "left_anterior_tibialis"=> ["lat", "l"],
-                                         "right_anterior_tibialis"=> ["rat", "r"]],
-                             ["heart_rate"] => ["heart_rate"=> ["hr"]],
-                             ["snore"] => ["snore"],
+                                         "left_anterior_tibialis"=> ["lat", "lat1", "l", "left", "leftlimb", "tibl", "tibli", "plml", "leg1", "lleg", "lleg1", "legl", "jambe_l", "leftleg"],
+                                         "right_anterior_tibialis"=> ["rat", "rat1", "r", "right", "rightlimb", "tibr", "tibre", "plmr", "leg2", "leg3", "rleg", "rleg1", "legr", "jambe_r", "rightleg"]],
+                             # it is common to see ambiguous channels, which could be leg or face EMG
+                             # if leg EMG is present in separate channels,
+                             # post-processing might map "emg_ambiguous"
+                             # to chin channels (for example)
+                             ["emg_ambiguous", "emg"] => ["1" => ["aux1", "l"],
+                                                          "2" => ["aux2", "r"],
+                                                          "3" => ["aux3"]],
+                             ["heart_rate"] => ["heart_rate"=> ["hr", "pulse", "pulso", "pr", "pulserate"]],
+                             ["snore"] => ["snore" => ["ronquido", "ronquido derivad", "schnarchen", "ronfl", "schnarchmikro"]],
```

Comment:
Ronquido, schnarchen, shnarchmikro... I'm going to bet these are some variant of "snore" in non-English languages. IMO other languages should be left for custom extractors, and we should probably document that we cover common English names only.

Reply:
My policy here has been to include alternative spellings / languages if they are un-ambiguous.